.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,16 @@
+<!--
+Thanks for contributing!
+-->
+
+Some things to make sure you have completed:
```

Comment:
Most of these tasks are deferred until merge time. For example, I know several folks prefer to delay JSON generation until the very end to cut down on the diff. Likewise, version bumps and dates often get placeholders until merge time.

```suggestion
Please complete the following before merging:
```